packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2137,6 +2137,25 @@ class _ElementLocationStatsTracker {
       }
       json['newLocations'] = locationsJson;
     }
+
+    // Add in a data structure for the location names.
+    if (newLocations.isNotEmpty) {
+      final Map<String, Map<int, String>> namesJson = <String, Map<int, String>>{};
+      for (final _LocationCount entry in newLocations) {
+        final _Location location = entry.location;
+        if (location.file != null) {
```

Comment:
as a follow up, location.file can now be non-nullable so you can delete this `!= null` check and others in the file.

Reply:
done